lib/analysis/modules/ies_enkf_data.c

```diff
@@ -32,14 +29,7 @@
 struct ies_enkf_data_struct {
    UTIL_TYPE_ID_DECLARATION;
    int       iteration_nr;            // Keep track of the outer iteration loop
-   double    ies_max_steplength;      // Maximum step length in Gauss Newton iteration
```

Comment:
Are we sure deleting these fields won't give us an ABI breakage wrt. stuff that exists in people's `storage/`?

Reply:
Are the structures dumped directly to disk?

Reply:
I may be misunderstanding you but I do not think any of the read/write mechanisms use `structs` directly to/from disk.